backtrack/anagram.py

```diff
@@ -31,14 +37,22 @@ def anagram(s1,s2):
         pos = ord(s2[i])-ord('a')
         c2[pos] = c2[pos] + 1
 
-    j = 0
-    stillOK = True
-    while j<26 and stillOK:
-        if c1[j]==c2[j]:
-            j = j + 1
-        else:
-            stillOK = False
+    return c1 == c2
+
+
+class TestSuite (unittest.TestCase):
+    def test_all_perms(self):
+        allPerms = ['abc', 'bac', 'bca', 'acb', 'cab', 'cba']
```

Comment:
The code should be formatted according to PEP guide.
Please use `snake_case` and avoid ` camelCase` for everything other than classes. Also there should be a space after each `,` and a newline before or after a function.